libraries/kotlin.test/common/src/main/kotlin/org/junit/Ignore.kt

```diff
@@ -0,0 +1,5 @@
+package org.junit
+
+@Deprecated("Use 'Ignore' from kotlin.test package", replaceWith = ReplaceWith("Ignore", imports = "kotlin.test.Ignore"))
+@Suppress("HEADER_WITHOUT_IMPLEMENTATION")
+header annotation class Ignore
```

Comment:
I think this is blocked by https://youtrack.jetbrains.com/issue/KT-19656.

I can't create `@Ignore` and alias to `org.junit.Ignore` without mirroring its full set of properties, despite the fact that on the JVM they all have defaults.